assets (1).py

```diff
@@ -0,0 +1,115 @@
+from datetime import UTC, datetime, timedelta
+from typing import Any
+from fastapi import FastAPI, HTTPException
+from sqlalchemy import DateTime, ForeignKey, create_engine, select
+from sqlalchemy.ext.asyncio import async_sessionmaker, create_async_engine
+from sqlalchemy.orm import DeclarativeBase, Mapped, Session, mapped_column, relationship, sessionmaker
+
+
+from mailing import AssetMailer
+
+# Assets belongs to parent and author
+# at any given time, one parent can only have one active asset
+
+
+class Asset(DeclarativeBase):
+    __tablename__ = "asset"
+
+    id: Mapped[int]
+    title: Mapped[str]
+    width: Mapped[int]
+    height: Mapped[int]
+    active: Mapped[bool]
+    created_at: Mapped[datetime] = mapped_column(DateTime(timezone=True))
+
+    parent_id: Mapped[int] = mapped_column(ForeignKey("parent.id"))
+    parent: Mapped["Parent"] = relationship(back_populates="assets")
+
+    author_id: Mapped[int] = mapped_column(ForeignKey("author.id"))
+    author: Mapped["Author"] = relationship(back_populates="assets")
```

Comment:
```suggestion
    author: Mapped["Author"] = relationship(back_populates="assets")
    def __repr__(self):
       return f"<Asset {self.id} - {self.title}>"
```

Je recommande des `__repr__` c'est de la qualite de vie gratuite pour debug plus facilement !